app/views/thesis/edit.html.erb

```diff
@@ -76,30 +79,25 @@
                               hint: '(Enter as: Last/Surname, First/Given name Middle name)',
                               hint_html: { class: 'col3q' } %>
 
-      <div class="layout-1q3q layout-band field-wrap">
-        <div class="col1q"><strong>Department(s)</strong></div>
-        <div class="col3q">
-          <% f.object.departments.find_all do |dept| %>
-            <p><%= dept.name_dw %></p>
-          <% end %>
+      <% f.object.departments.find_all do |dept| %>
+        <% dept_id = f.object.departments.find_index(dept).to_i+1 %>
+        <div class="layout-1q3q layout-band field-wrap">
+          <label class="col1q field-label" for="department-<%= dept_id %>">Department <%= dept_id %></label>
```

Comment:
Could we only display the dept_id here if it's greater than 1? That way most users would just read Department but if there are multiple it would still read as Department 2?

Reply:
Yeah, that should be doable (and result in a more logical display for users). This probably makes sense to do for both degree and department fields. I'll push a new commit.